pkg/storage/tsdb/config.go

```diff
@@ -199,6 +204,9 @@ func (cfg *TSDBConfig) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.MemorySnapshotOnShutdown, "blocks-storage.tsdb.memory-snapshot-on-shutdown", false, "True to enable snapshotting of in-memory TSDB data on disk when shutting down.")
 	f.IntVar(&cfg.HeadChunksWriteQueueSize, "blocks-storage.tsdb.head-chunks-write-queue-size", 0, "The size of the write queue used by the head chunks mapper. Lower values reduce memory utilisation at the cost of potentially higher ingest latency. Value of 0 switches chunks mapper to implementation without a queue.")
 	f.BoolVar(&cfg.IsolationEnabled, "blocks-storage.tsdb.isolation-enabled", false, "[Deprecated] Enables TSDB isolation feature. Disabling may improve performance.")
+	f.DurationVar(&cfg.OOOAllowance, "blocks-storage.tsdb.ooo-allowance", 0*time.Second, "Allow upto this much out-of-order.  Supported units: h, m, s.")
```

Comment:
"upto this much" does not tell what it is, it can be number of series, sample rate, etc. Suggesting update in description.

```suggestion
	f.DurationVar(&cfg.OOOAllowance, "blocks-storage.tsdb.ooo-allowance", 0, "Allow samples to be this old for out-of-order.  Supported units: h, m, s.")
```